templates/page.tmpl

```diff
@@ -20,7 +20,7 @@
         </TMPL_IF>
 	<div id="center-logo">
 	  <a href="<TMPL_VAR BASEURL>index.html" title="Back to main page">
-	    <img alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome-logo.svg">
+	    <img alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome-logo-1.png">
```

Comment:
> The change provides possibilities for mobiles (small screen) and large 4K screens.

replacing svg pic to png is def the opposite

Reply:
@raven2cz ah, and one more thing about logo - these gray ovals in the right corner looking a bit out of place

Reply:
> these gray ovals in the right corner

That's the Lua logo.